src/components/App/App.js

```diff
@@ -21,7 +21,7 @@ class App extends Component {
         email: '',
         purpose: ''
       },
-      isLoggedIn: false,
+      isLoggedIn: true,
```

Comment:
This is just to make it easier while working in the app, will be set to false when the project is turned in.

Reply:
Sounds good!